### PR TITLE
ci for uenv testing on santis

### PR DESCRIPTION
**Pull request by jgphpc**
see https://gitlab.com/cscs-ci/ci-testing/webhook-ci/mirrors/6557018579987861/835515875116204/-/jobs (squash before merge)

### CODE REVIEW

File: config/systems-uenv/eiger-uenv.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+            'target_systems': ['eiger'],
+        }
+    ],
+    'environments': actual_environs,
+    'general': [
+        {
+             # 'resolve_module_conflicts': False,
+             'target_systems': ['eiger']
+        }
+    ]
```
Comment (ekouts) [inline config/systems-uenv/eiger-uenv.py:153]: Let's remove this section if we are not using it?

File: config/systems-uenv/eiger-uenv.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        ],
+        'max_jobs': 100,
+        'extras': {
+            'cn_memory': 500,
+        },
+        'access': [
+            '-pnormal',
+            '-Cmc',
+            f'--account={osext.osgroup()}'
+        ] + uenv_access,
```
Comment (ekouts) [inline config/systems-uenv/eiger-uenv.py:76]: This doesn't need to be in the partitions anymore, we can put it in the environment. You can see an example in the description of this PR: https://github.com/reframe-hpc/reframe/pull/3152
What do you think?

File: config/systems-uenv/eiger-uenv.py [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    env['prepare_cmds'] = [
+        'myview=$(uenv --no-color view |grep -m1 "uenv view")',
+        'echo myview=$myview',
+        'eval $myview',
+        # --- workaround for https://jira.cscs.ch/browse/VCUE-236:
+        f'export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$UENV_MOUNT_POINT/env/$(echo $UENV_VIEW |cut -d: -f3)/lib'
+        # f'source {activation_script}',
+    ]
+    env['name'] = f'{image_name}_{k}'
+    del env['activation']
```
Comment (ekouts) [inline config/systems-uenv/eiger-uenv.py:117]: I am a bit confused, where is this set and why do you delete it?

File: ci/alps_uenv.yml [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+run reframe:
+  stage: run
+  tags: ['f7t-runner']
+  variables:
+    GIT_STRATEGY: "fetch"
+    SLURM_PARTITION: "debug"
+    SLURM_CONSTRAINT: "mc"
+    F7T_TOKEN_URL: "https://auth.cscs.ch/auth/realms/firecrest-clients/protocol/openid-connect/token"
+    F7T_URL: "https://firecrest.cscs.ch"
```
Comment (ekouts) [inline ci/alps_uenv.yml:67]: Let's clean up these variables, right?

File: ci/scripts/alps_uenv.sh [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+uenv_sqfs_fullpath() {
+    img="$1"
+    uenv image inspect $img --format {sqfs}
+}   
+# }}}
+# {{{ launch_reframe 
+launch_reframe() {
+    img=$1
+    # export UENV="${squashfs_path}:${mount}"
+    export UENV="$img"
```
Comment (jgphpc) [inline ci/scripts/alps_uenv.sh:191]: I will use here the image name (prgenv-gnu/xx, etc...) instead of store.sqfs

**Comment by jgphpc** [comment]
Example ci: https://gitlab.com/cscs-ci/ci-testing/webhook-ci/mirrors/6557018579987861/835515875116205/-/jobs/6771838264

File: config/systems-uenv/eiger-uenv.py (hunk 3 above)
Comment (jgphpc) [reply config/systems-uenv/eiger-uenv.py:117]: @teojgo ?

**Pull request merged**